web/src/components/core/ProgressText.jsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) [2023] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+// @ts-check
+
+import React from "react";
+import { Spinner, Text } from "@patternfly/react-core";
+
+/**
+ * Progress description
+ *
+ * @component
+ *
+ * @param {object} props
+ * @param {string} [props.message] Progress message
+ * @param {number} [props.current] Current step
+ * @param {number} [props.total] Number of steps
+ */
+export default function ProgressText({ message, current, total }) {
+  const text = (current === 0) ? message : `${message} (${current}/${total})`;
+  return (
+    <div className="split">
+      <Spinner size="md" aria-label="Progress indicator" />
```

Comment:
I know we were talking about this, but maybe worth it to test how it looks with `<Icon name="loading" size="24" />` (in fact, I already did it) just for the sake of using just one "spinner" across the app. Although, as we discussed, we can distinguish between "Loading page" and "Loading state" and keep the consistence accordingly.

| Spinner | Loading Icon |
|:-:|:-:|
| ![spinner](https://user-images.githubusercontent.com/1691872/217763699-c8711fb9-3a33-40c3-aacd-6978e639b7f2.gif) | ![loading](https://user-images.githubusercontent.com/1691872/217763740-ea1677b5-b1f6-4931-9084-2b98d723a86d.gif) |